### DISCUSSION TITLE
Product Image Gallery not showing in Quick View

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#167] [1. Help]
Hi,

I have multiple images of a product which shows in correct form on PDP page. 
![image](https://github.com/breezefront/community/assets/132253509/75b751bd-d466-47ad-abfa-aaeca5c908d4)

But on listing page when we use Quick View option it shows only one main image. Other Images not showing.
![image](https://github.com/breezefront/community/assets/132253509/d980fad1-660c-48da-a080-3ef1a303d3ca)

Any possible fix?

**Comment by vovayatsyuk** [comment]
What module do you use for QuickView?

Works fine with AjaxPro: https://argento-breeze-m2.swissupdemo.com/force_breeze/women-category/clothing/jackets.html

**Reply by dheerajKsharma** [reply]
I am using the same module. Swissup_AjaxPro

**Reply by vovayatsyuk** [reply]
Are your modules up to date?

**Reply by dheerajKsharma** [reply]
Yes, All modules are up to date.
![image](https://github.com/breezefront/community/assets/132253509/dc159172-ba43-4c5c-afc7-e6e90716cd3d)

**Reply by vovayatsyuk** [reply]
Please provide a link to your store

**Reply by dheerajKsharma** [reply]
Listpage: https://artuk.wcltest.com/prints.html
PDP: https://artuk.wcltest.com/john-lavery-spring-the-garden-villa-sylvia-coaster.html

**Reply by vovayatsyuk** [reply]
Looks like you need to install `swissup/module-breeze-integrations` package. It has integrations with a few Swissup modules.

**Reply by dheerajKsharma** [reply]
I think I already have it.
![image](https://github.com/breezefront/community/assets/132253509/94f855f9-c0fa-4431-bb97-3bbe73738a77)

**Reply by vovayatsyuk** [reply]
That's weird. Are you sure this module is enabled (bin/magento module:enable)? 

If so, please create a ticket at swissuplabs site with ssh access to your store.

**Reply by dheerajKsharma** [reply]
Can it be this error related to the cause of this issue?
![image](https://github.com/breezefront/community/assets/132253509/03a74198-c3a1-4004-8889-0b79f451b598)

**Reply by vovayatsyuk** [reply]
It's not working correctly because fotorama is used inside Ajaxpro popup. It should not. `swissup/module-breeze-integrations` should prevent this.

**Reply by dheerajKsharma** [reply]
Hi @vovayatsyuk ,
I have new finding in this. When I am excluding /customer in Breeze Exclude section in admin configuration. It disabled gallery. When I remove it, it works well.
![image](https://github.com/breezefront/community/assets/132253509/eeda25e9-cb4b-45c5-b53f-fb2383fc4cc1)

Thanks

**Reply by vovayatsyuk** [reply]
Oh, I see. The issue happens because this popup is loaded via `customer/section/load` URL.

I'm thinking to extend this option for exceptions. For example:

```
/customer,-customer/section/load,-customer/another/page
```

The code above will disable `customer` URLs unless it's a `customer/section/load` or `customer/another/page`.

---

On the other hand, the `customer/section/load` URL must return Breeze content if requested from Breeze page, so I will just add it to the hardcoded exceptions somewhere in the code.

I will let you know when it will be fixed.

**Reply by vovayatsyuk** [reply]
Fixed in https://github.com/breezefront/module-breeze/commit/c91233cb4450a2b33a06aaa68432944af55499c3

**Reply by parthvisavadiya** [reply]
In this there is one more issue, when on PDP we open quickview popup and close it again, the quickview's product image stick on pdp. If you have resolved it please share what you have done for that.

**Reply by vovayatsyuk** [reply]
@parthvisavadiya looks like it's another issue, please open new discussion.
